packages/@uppy/core/src/index.js

```diff
@@ -556,6 +556,13 @@ class Uppy {
       files: updatedFiles
     })
 
+    // If nothing is uploading anymore, and if nothing has uploaded yet - allow new uploads!
+    if (Object.keys(updatedFiles).length === 0) {
+      this.setState({
+        allowNewUpload: true
```

Comment:
Could we combine this with the `setState` call above, so we don’t have to call it twice?

Reply:
Agreed, pushed a new commit!